### PR TITLE
add driverTest + fix driver typo

### PR DESCRIPTION
**Pull request by khecova**

### CODE REVIEW

File: src/main/java/cz/muni/pa165/teamwhite/formula1/entity/Driver.java [hunk 1]
```diff
@@ -85,8 +85,8 @@ public String getNationality() {
         return nationality;
     }
 
-    public boolean isAgresive() {
-        return isAgresive;
+    public boolean isAggressive() {
```
Comment (Jakub-Fajkus) [inline src/main/java/cz/muni/pa165/teamwhite/formula1/entity/Driver.java:88]: :heart:

File: src/test/java/cz/muni/pa165/teamwhite/formula1/DriverTest.java [hunk 2]
```diff
@@ -0,0 +1,266 @@
+package cz.muni.pa165.teamwhite.formula1;
```
Comment (Jakub-Fajkus) [inline src/test/java/cz/muni/pa165/teamwhite/formula1/DriverTest.java:1]:
```suggestion
package cz.muni.pa165.teamwhite.formula1.dao;
```

Move to .dao package

**Review by Jakub-Fajkus — APPROVED** [review]
Just move the test to .dao package. Otherwise, it's perfect for me :+1: Even `mvn clean install` ends successfully :heart:

File: src/test/java/cz/muni/pa165/teamwhite/formula1/DriverTest.java (hunk 2 above)
Comment (khecova) [reply src/test/java/cz/muni/pa165/teamwhite/formula1/DriverTest.java]: fixed

**Review by tomasedl — APPROVED** [review]
nice :)

**Review by andrlos — APPROVED** [review]
well done!

**Pull request merged**